test/fixtures/e2e_suite.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright 2023 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fixtures
```

Comment:
I see an opportunity of abstraction. Should we create a util repository to hold the E2E testing fixtures such that all UDFs/UDSource/Sink etc. can share? cc: @vigith @whynowy

Reply:
we should extract one out for contrib org.